.github/workflows/ci.yml

```diff
@@ -99,8 +99,9 @@ jobs:
             args: check --locked
           - command: test
             args: --all-features --workspace
-          - command: test
```

Comment:
We need to at least add a cargo check at the workspace level otherwise running `cargo check` in the repo can break (which breaks IDEs and is just bad practice)

Reply:
this only affected test, we have a `cargo check` job 4 lines above this.